lib/middleware/cordova/cordova_plugins.js

```diff
@@ -11,17 +11,22 @@ var fs = require('fs'),
  * Serves the cordova_plugins.js file whenever it is missing.
  */
 
-module.exports = function() {
+module.exports = function(options) {
     // return the request listener
     return function(req, res, next) {
         if (req.url.indexOf('cordova_plugins.js') >= 0) {
-            var filepath = path.join(
-                __dirname,
-                '../../../res/middleware/cordova',
-                req.session.device.version,
-                req.session.device.platform,
-                'cordova_plugins.js'
-            );
+			var filepath;
```

Comment:
I believe this repository was using 4-space tabs and you're using tab-character tabs. I think we use 4-space in our other repositories.